Shaders/FilmicAnamorphSharpen.fx

```diff
@@ -0,0 +1,180 @@
+/*
+Filmic Anamorph Sharpen PS (c) 2018 Jacob Maximilian Fober
+
+This work is licensed under the Creative Commons 
+Attribution-ShareAlike 4.0 International License. 
+To view a copy of this license, visit 
+http://creativecommons.org/licenses/by-sa/4.0/.
+*/
+
+  ////////////////////
+ /////// MENU ///////
+////////////////////
+
+uniform float Strength <
+	ui_label = "Sharpen strength";
+	ui_type = "drag";
+	ui_min = 0.0; ui_max = 3.0; ui_step = 0.005;
+> = 1.0;
+
+uniform bool Preview <
+	ui_label = "Preview";
+	ui_tooltip = "Preview sharpen layer and mask for adjustment";
+> = false;
+
+uniform int Coefficient <
+	ui_label = "Luma coefficient";
+	ui_tooltip = "Change if objects with relatively same brightness but different color get sharpened";
+	ui_type = "combo";
+	ui_items = "BT.709\0BT.601\0";
+> = 0;
+
+uniform float Clamp <
+	ui_label = "Sharpen clamping";
+	ui_type = "drag";
+	ui_min = 0.5; ui_max = 1.0; ui_step = 0.001;
+> = 1.0;
+
+uniform int Offset <
+	ui_label = "High-pass offset";
+	ui_tooltip = "High-pass cross offset in pixels";
+	ui_type = "drag";
+	ui_min = 0; ui_max = 2;
+> = 1;
+
+uniform int Contrast <
+	ui_label = "Edges mask";
+	ui_tooltip = "Depth high-pass mask amount";
+	ui_type = "drag";
+	ui_min = 0; ui_max = 2000; ui_step = 1;
+> = 1618;
+
+  //////////////////////
+ /////// SHADER ///////
+//////////////////////
+
+#include "ReShade.fxh"
+
+// Overlay blending mode
+float Overlay(float LayerA, float LayerB)
+{
+	float MinA = min(LayerA, 0.5) * 2;
+	float MinB = min(LayerB, 0.5) * 2;
+
+	float MaxA = 2 - max(LayerA, 0.5) * 2;
+	float MaxB = 2 - max(LayerB, 0.5) * 2;
+
+	return (MinA * MinB + 1 - MaxA * MaxB) * 0.5;
+}
+
+// Convert RGB to YUV.luma
+float Luma(float3 Source, float3 Coefficients)
+{
+	float3 Result = Source * Coefficients;
+	return Result.r + Result.g + Result.b;
+}
+
+// Define screen texture with mirror tiles
+texture TexColorBuffer : COLOR;
+sampler SamplerColor
+{
+	Texture = TexColorBuffer;
```

Comment:
It's easier if you just write `Texture = ReShade::BackBufferTex` here, since it was already defined in ReShade.fxh and you don't need `TexColorBuffer`.

Reply:
Fixed